client/packages/invoices/src/Prescriptions/api/operations.graphql

```diff
@@ -2,7 +2,7 @@ fragment PrescriptionRow on InvoiceNode {
   __typename
   comment
   createdDatetime
-  prescriptionDate: allocatedDatetime
+  prescriptionDate: backdatedDatetime
```

Comment:
Kept this as prescriptionDate to minimise changes in frontend, I think it's ok how it is but maybe it's better to make it clearly backdatedDatetime in frontend?

Reply:
Yeah I think backdateDatetime would be better for front end